tests/console/rmt/rmt_proxy.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: yast2-rmt rmt-server
+# Summary: Add rmt configuration test and basic configuration via
+#    rmt-wizard, test RMT basic function behind proxy.
+# Maintainer: Lemon Li <leli@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use repo_tools;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    record_info('RMT server setup', 'Start to setup a rmt server');
+    rmt_wizard();
+    # sync from SCC
+    rmt_sync;
+
+    # Set RMT behind a proxy (with NTLM Auth), and ensure RMT basic function works.
+    record_info('Set proxy', 'Set RMT behind a proxy (with NTLM Auth)');
+    # set the proxy setting to rmt.conf
+    my $proxy_url = 'http:\/\/' . get_required_var('PROXY_SERVER');
+    my $rmt_conf = '/etc/rmt.conf';
+    my $rmt_conf_bk = '/etc/rmt.conf.bk';
+
+    # backup the rmt conf file
+    assert_script_run("cp $rmt_conf $rmt_conf_bk");
+
+    # update the rmt conf file with the proxy setting
+    assert_script_run("sed -i 's/proxy: /proxy: $proxy_url/g' $rmt_conf");
+    assert_script_run("sed -i 's/proxy_auth: /proxy_auth: ntlm/g' $rmt_conf");
+    assert_script_run("systemctl restart rmt-server");
+
+    # verify the basic function of the rmt server
+    rmt_sync;
+    assert_script_run("rmt-cli product enable SLES/12.5/x86_64");
```

Comment:
Is there any other way to confirm rmt still work instead of hardcode some product name?

Reply:
Maybe you know other method to do this? In fact this is the automation for manual test, just follow the requirement from manual test steps.

Reply:
Is there any simple rmt-cli command to just confirm connection with scc ? For example check the connection status of scc, not enable some specific product. We should try best avoid this hardcode

Reply:
I haven't found method to use enable to verify but avoid the hardcode, and the verification through rmt_cli enable is required from Chenzi.

Reply:
If we still need check product enable operation , i suggest we can get the product name by output of rmt-cli instead of hardcode.